src/sentry/api/endpoints/organization_events_trace.py

```diff
@@ -81,6 +82,18 @@ def construct_span_map(self, events, key):
                 parent_map[item[key]].append(item)
         return parent_map
 
+    def check_missing_service(self, child, domain, event):
+        # Skip the method prefix on http spans
+        child_domain = child["description"].split()[-1]
+        # string contains is significantly faster than urlparse
+        if (
+            domain in child_domain and urlparse(child_domain).netloc == domain
```

Comment:
`urlparse('https://sentry.io/organizations/test/performance/').netloc`
100000 loops, best of 3: 2.74 usec per loop
vs
`'sentry.io' in 'https://sentry.io/organizations/test/performance/'`
10000000 loops, best of 3: 0.058 usec per loop